src/main/java/com/epam/ta/reportportal/commons/querygen/FilterTarget.java

```diff
@@ -273,6 +273,11 @@ protected void joinTables(SelectQuery<? extends Record> query) {
 					.withJoinCondition(LAUNCH.ID.eq(ITEM_ATTRIBUTE.LAUNCH_ID))
 					.get(),
 
+			new CriteriaHolderBuilder().newBuilder(CRITERIA_ITEM_ATTRIBUTE_SYSTEM, ITEM_ATTRIBUTE.SYSTEM, Boolean.class)
```

Comment:
We can't use it for filtering.
Maybe we need to find a way how to ignore system attributes when aggregating them into an array